jena-fuseki2/apache-jena-fuseki/assembly-dist.xml

```diff
@@ -103,7 +103,7 @@
      </fileSet>
 
     <fileSet>
-      <directory>../jena-fuseki-webapp/src/main/webapp</directory>
+      <directory>../jena-fuseki-webapp/target/webapp/</directory>
```

Comment:
The `target` contains the `jena-fuseki-webapp` files, plus the dist files of `jena-fuseki-ui` (see `webapp`'s `pom.xml`).